### PR TITLE
Support schedule event; expose event icon

### LINKED ISSUE #21: Support schedule event

### LINKED ISSUE #22: Expose event type
Perhaps a subtle [Octicon](https://primer.style/octicons/) in the context block

### PR DESCRIPTION
**Pull request by namoscato**
Resolves #21, resolves #22 

* Rename `githubClient` ➡️ `octokit` 
* Add [Octicon](https://primer.style/octicons/) image representing event type to beginning of context
* Support `schedule` event type

### CODE REVIEW

File: src/github/context.ts [hunk 1]
```diff
@@ -2,8 +2,21 @@ import {context} from '@actions/github'
...
+  UnsupportedEventError
+} from './webhook'
+
+export const EVENT_NAME_IMAGE_MAP: Record<SupportedEventName, string> = {
+  pull_request:
+    'https://user-images.githubusercontent.com/847532/193414326-5aaf5449-0c81-4a66-9b19-4e5e6baeee9e.png',
+  push: 'https://user-images.githubusercontent.com/847532/193413878-d5fcd559-401d-4954-a44c-36de5d6a7adf.png',
+  schedule:
+    'https://user-images.githubusercontent.com/847532/193414289-3b185a3b-aee8-40f9-99fe-0615d255c8dd.png'
+} as const
```
Comment (namoscato) [inline src/github/context.ts:19]: Screen Shot 2022-10-02 at 9 54 45 AM

Screen Shot 2022-10-02 at 9 54 56 AM

I added a semi-transparent stroke such that these are somewhat visible in dark mode, but they're optimized for light mode.

File: src/github/summary.ts [hunk 2]
```diff
@@ -81,18 +92,34 @@ function getEventLink(): Link {
...
+    }
+  }
+
+  if (isScheduleEvent(context)) {
+    const commit = (
+      await octokit.rest.repos.getCommit({
+        ...context.repo,
+        ref: context.sha
+      })
+    ).data.commit
```
Comment (namoscato) [inline src/github/summary.ts:114]: The `schedule` event payload doesn't have much – mostly just repository information – so we have to fetch the commit.

**Comment by johnkoehn** [comment]
O geez, my bad -- taking look now 👀

File: src/github/context.ts (hunk 1 above)
Comment (johnkoehn) [reply src/github/context.ts:19]: wow, I love this image idea!

Comment (johnkoehn) [reply src/github/context.ts:19]: Where are you getting these url from?

**Review by johnkoehn — APPROVED** [review]

File: src/github/context.ts (hunk 1 above)
Comment (namoscato) [reply src/github/context.ts:19]: I uploaded them into some GitHub markdown comment and hope they'll just live there indefinitely. Do you have concerns or suggestions on a better place to host them?

Comment (johnkoehn) [reply src/github/context.ts:19]: O I see. That seems like the best strategy to keep this working indefinitely

**Pull request merged**